src/components/admin/lib/article-validators.js

```diff
@@ -0,0 +1,21 @@
+export function hasNonHttpsURL(html) {
+  // if (/http:\/\/[\S]+\.[a-zA-Z]{2,}/.test(html) || /http(?!s)/.test(html)) {
+  if (/<a.*?href\s*?=\s*?["'](http(?!s))(.*?)["'] *?>/.test(html)) {
```

Comment:
There is no reason for the extra capture group here around `http(?!s)` as we aren't capturing anything, the same for around `.*?`